src/Parser/Warlock/Affliction/Modules/Features/Checklist.js

```diff
@@ -73,23 +67,15 @@ class Checklist extends CoreChecklist {
         ];
       },
     }),
+    // TODO: figure out what to do with this rule (only Haunt survived and it may not get picked a lot over Shadow Embrace)
     new Rule({
       name: <React.Fragment>Buff your <SpellLink id={SPELLS.UNSTABLE_AFFLICTION_CAST.id} /> as much as possible</React.Fragment>,
-      description: <React.Fragment><SpellLink id={SPELLS.UNSTABLE_AFFLICTION_CAST.id} /> is your biggest source of damage and you should try to buff its damage as much as possible with <SpellLink id={SPELLS.REAP_SOULS.id} /> and <SpellLink id={SPELLS.MALEFIC_GRASP_TALENT.id} /> or <SpellLink id={SPELLS.HAUNT_TALENT.id} /> (if talented). <br />
-        If you don't have <SpellLink id={SPELLS.WARLOCK_TORMENTED_SOULS.id} />, it's ok to wait a bit (as long as you're not wasting Soul Shards while waiting) and cast Unstable Affliction when you can buff it with Reap Souls.
+      description: <React.Fragment><SpellLink id={SPELLS.UNSTABLE_AFFLICTION_CAST.id} /> is your biggest source of damage and you should try to buff its damage as much as possible with Reap Souls and Malefic Grasp or <SpellLink id={SPELLS.HAUNT_TALENT.id} /> (if talented). <br />
+        If you don't have Tormented Souls, it's ok to wait a bit (as long as you're not wasting Soul Shards while waiting) and cast Unstable Affliction when you can buff it with Reap Souls.
```

Comment:
I'm guessing this line doesn't make sense anymore with the artifact ability being removed?

Reply:
Yeah, I mentioned that in the description of this PR, that I need to revisit the texts. For now it was just to fix it from crashing.